providers/default.rb

```diff
@@ -438,7 +438,7 @@ def execute_install_script(install_script)
       action :nothing
     end.run_action(:run)
   else
-    upgrade_command = Mixlib::ShellOut.new(install_script)
+    upgrade_command = Mixlib::ShellOut.new(install_script, { :timeout => new_resource.install_timeout } )
```

Comment:
Need a chefstyle fix I believe, I'm assuming this should be `timeout: new_resource.install_timeout`

Reply:
Oh, probably; I was looking at other Mixlib::ShellOut examples in the wild to make sure I was doing it right, and they use ye olde hashrockets; I'll update to the modern syntax.